### ISSUE TITLE
how to set the arguments for find_learning_rate() function to find best learning rate?

### ISSUE THREAD

**Issue by cpoptic** [flairNLP/flair#700]
My corpus is relatively small (~130 sentences), so I'm wondering how to set the parameters of the find_learning_rate method to actually find the ideal LR?

learning_rate_tsv = trainer.find_learning_rate(BASE_DIR,
                                       'learning_rate.tsv',
                                       iterations = 100,)

Should I use the default start_learning_rate and end_learning_rate?
How about the mini_batch_size?  With a small number of sentences in the corpus at ~130, of which 70% are in the training set, should I adjust the batch size?
What about the number of iterations?  Keep at default of 100?
How about the smoothing_factor?

**Comment by alanakbik** [comment]
Good question, but with a corpus this small I'm not sure if the method will work. @kashif do you have any intuition here?
